### PR TITLE
updated comparison 

### PR DESCRIPTION
**Pull request by rkbright**
Howdy John, this might be throw away work but wanted to share. I switched a few of the test comparisons and placed the ``want`` before ``got``.

I also forgot how you wanted to review these..did you first want to review in my forked repo?

### CODE REVIEW

**Comment by bitfield** [comment]
Hey @rkbright, thanks for this! It looks like your fork might be a little out of date—the PR is reporting conflicts that need to be resolved. Could you take a look at those, and then I'll review this for you.

**Comment by rkbright** [comment]
Sorry about that, I resolved the two (2) issues. I see you updated your README with links to your recent books - awesome!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ A library is easier to use, and easier for contributors to work on, if it has a
 
 It goes without saying, but I'll say it anyway, that you must provide comprehensive tests for your feature. Code coverage doesn't need to be 100% (that's a waste of time and effort), but it does need to be very good. The [awesome-go](https://github.com/avelino/awesome-go) collection (which `script` is part of) mandates at least 80% coverage, and I'd rather it were 90% or better.
 
-Test data should go in the `testdata` directory. If you create a file of data for input to your method, name it `method_name.input.txt`. If you create a 'golden' file (of correct output, to compare with the output from your method) name it `method_name.golden.txt`. This will help keep things organised.
+Test data should go in the `testdata` directory. If you create a file of data for input to your method, name it `method_name.input.txt`. If you create a 'golden' file (of correct output, to compare with the output from your method) name it `method_name.golden.txt`. This will help keep things organized.
```
Comment (bitfield) [inline CONTRIBUTING.md:75]: I'm British, and so is my spelling!

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 [![GoDoc](https://godoc.org/github.com/bitfield/script?status.png)](http://godoc.org/github.com/bitfield/script)[![Go Report Card](https://goreportcard.com/badge/github.com/bitfield/script)](https://goreportcard.com/report/github.com/bitfield/script)[![Mentioned in Awesome Go](https://awesome.re/mentioned-badge-flat.svg)](https://github.com/avelino/awesome-go)[![CircleCI](https://circleci.com/gh/bitfield/script.svg?style=svg)](https://circleci.com/gh/bitfield/script)
 
+# How do I import it?
```
Comment (bitfield) [inline README.md:3]: This isn't a bad idea, but I think on balance I prefer it the way it is.

File: CONTRIBUTING.md (hunk 1 above)
Comment (rkbright) [reply CONTRIBUTING.md:75]: For sure, and to think, I should have know better having lived in Europe for several years.

File: README.md (hunk 2 above)
Comment (rkbright) [reply README.md:3]: Yup, I completely understand.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bitfield) [reply CONTRIBUTING.md:75]: 😆

**Pull request closed without merging**